### PR TITLE
MAINT: Fix linkcheck

### PR DESCRIPTION
**Pull request by larsoner**
Should fix https://app.circleci.com/pipelines/github/mne-tools/mne-python/21586/workflows/ff60e63b-5cf1-4a53-bb74-35f154cec835/jobs/60863

### CODE REVIEW

File: doc/development/contributing.rst [hunk 1]
```diff
@@ -215,7 +215,7 @@ Once you have git installed and configured, and before creating your local copy
 of the codebase, go to the `MNE-Python GitHub`_ page and create a `fork`_ into
 your GitHub user account.
 
-.. image:: https://docs.github.com/assets/cb-28613/images/help/repository/fork_button.png
+.. image:: https://docs.github.com/assets/cb-79331/mw-1440/images/help/repository/fork-button.webp
```
Comment (drammock) [inline doc/development/contributing.rst:219]: the new image is really different from the prior one. I think we should just delete the image, it doesn't add a lot.

```suggestion
```

File: mne/cov.py [hunk 2]
```diff
@@ -1895,9 +1895,8 @@ def regularize(
     .. note:: This function is kept for reasons of backward-compatibility.
               Please consider explicitly using the ``method`` parameter in
               :func:`mne.compute_covariance` to directly combine estimation
-              with regularization in a data-driven fashion. See the `faq
-              <http://mne.tools/dev/overview/faq.html#how-should-i-regularize-the-covariance-matrix>`_
-              for more information.
+              with regularization in a data-driven fashion. See the
+              `FAQ <faq_how_should_i_regularize_>`_ for more information.
```
Comment (drammock) [inline mne/cov.py:1899]: shouldn't this be a `:ref:` not a link?

**Comment by larsoner** [comment]
Pushed the suggested corrections!

**Review by drammock — APPROVED** [review]

**Comment by drammock** [comment]
oh the irony!
https://app.circleci.com/pipelines/github/mne-tools/mne-python/21592/workflows/dfbb9e66-ece3-4e44-9a41-312ec8ceb656/jobs/60875?invite=true#step-109-53_214

**Pull request merged**